sdk/python/tests/test_client.py

```diff
@@ -324,687 +299,182 @@ def test_version(self, mocked_client, mocker):
+        ft1 = FeatureTable(
+            name="my-feature-table-1",
+            features=[
+                Feature(name="fs1-my-feature-1", dtype=ValueType.INT64),
+                Feature(name="fs1-my-feature-2", dtype=ValueType.STRING),
+                Feature(name="fs1-my-feature-3", dtype=ValueType.STRING_LIST),
+                Feature(name="fs1-my-feature-4", dtype=ValueType.BYTES_LIST),
+            ],
+            entities=["fs1-my-entity-1"],
+            labels={"team": "matchmaking"},
+            batch_source=batch_source,
+            stream_source=stream_source,
         )
 
-    def test_stop_ingest_job(self, mock_jobcontroller_client, mocker):
-        mocker.patch.object(
-            mock_jobcontroller_client,
-            "_jobcontroller_service_stub",
-            return_value=Core.JobControllerServiceStub(grpc.insecure_channel("")),
-        )
+        # Register Feature Table with Core
+        test_client.apply_feature_table(ft1)
 
-        ingest_job = IngestJob(
-            job_proto=IngestJobProto(
-                id="kafka-to-redis",
-                external_id="job#2222",
-                status=IngestionJobStatus.RUNNING,
-            ),
-            core_stub=mock_jobcontroller_client._jobcontroller_service_stub,
-        )
+        feature_tables = test_client.list_feature_tables()
 
-        mock_jobcontroller_client.stop_ingest_job(ingest_job)
+        # List Feature Tables
         assert (
-            mock_jobcontroller_client._jobcontroller_service_stub.StopIngestionJob.called
+            len(feature_tables) == 1
+            and feature_tables[0].name == "my-feature-table-1"
+            and feature_tables[0].features[0].name == "fs1-my-feature-1"
+            and feature_tables[0].features[0].dtype == ValueType.INT64
+            and feature_tables[0].features[1].name == "fs1-my-feature-2"
+            and feature_tables[0].features[1].dtype == ValueType.STRING
+            and feature_tables[0].features[2].name == "fs1-my-feature-3"
+            and feature_tables[0].features[2].dtype == ValueType.STRING_LIST
+            and feature_tables[0].features[3].name == "fs1-my-feature-4"
+            and feature_tables[0].features[3].dtype == ValueType.BYTES_LIST
+            and feature_tables[0].entities[0] == "fs1-my-entity-1"
         )
 
     @pytest.mark.parametrize(
-        "mocked_client",
-        [
-            lazy_fixture("mock_client"),
-            lazy_fixture("mock_client_with_auth"),
-            lazy_fixture("secure_mock_client"),
-            lazy_fixture("secure_mock_client_with_auth"),
-        ],
+        "mocked_client", [lazy_fixture("mock_client")],
     )
-    def test_get_historical_features(self, mocked_client, mocker):
-
-        mocked_client._serving_service_stub = Serving.ServingServiceStub(
-            grpc.insecure_channel("")
-        )
+    def test_ingest(self, mocked_client, mocker):
```

Comment:
can we please have all cases coverage?
w / wo partition column
pandas / file input
file / bq source